app/component/SummaryPage.js

```diff
@@ -2653,7 +2653,7 @@ const SummaryPageWithStores = connectToStores(
   ['MapLayerStore'],
   ({ getStore }) => ({
     mapLayers: getStore('MapLayerStore').getMapLayers({
-      notThese: ['stop', 'citybike', 'vehicles'],
+      notThese: ['stop', 'vehicles'],
```

Comment:
@leonardehrenfried  Looks like stops and vehicles (Bus positions) are also hidden if a route is displayed on the map. Is this the intended behavior?

Reply:
It seems so but to be honest I haven't been able to figure out what the logic or overarching vision is behind switching these on or off.

Reply:
Ok. The issue only asked for fixing missing sharing pois, so I fixed that. Tell me if you think we should also show stop and vehicle markers on the map, if a route is displayed.

Reply:
my 2ct comment on this: 
1. it would be useful to display intermediate (bus) stops on the displayed route as smaller circles (milestones) on the way to the end of that modal route (where the current bigger circle is displayed already). 
2. I consider carpooling meeting points (P&M stops) as "sharing POIs" too. Not sure if this is implemented this way.